frontend/src/components/MyProfile/MyProfile.vue

```diff
@@ -0,0 +1,104 @@
+<script setup>
+import DeleteIcon from '@/assets/img/delete_icon.svg';
+import ProfileIllustration from '@/assets/img/illustrations/profile_illustration.svg';
+import { useRouter } from 'vue-router';
+import userService from '@/api/users.js';
+import { useUserStore } from '@/stores/user';
+
+const userStore = useUserStore();
+const router = useRouter();
+const logout = () => {
+  userStore.logoutUser();
+  router.push({ name: 'auth' });
+};
+const deleteUser = async () => {
+  const isDeleteConfirmed = confirm('Do you really want to delete account?');
+  if (isDeleteConfirmed === false) return;
```

Comment:
maybe something like this would be a bit more elegant:
```js
if (!confirm('Are you sure?')) {
  return;
}
```

Reply:
I separated confirm as new variable for readability, in my opinion it's more readable that way.  Changed condition check to logical not so it's consistent.